assets/js/components/login/Logout.tsx

```diff
@@ -10,7 +10,7 @@ const LogoutButton: React.FunctionComponent = () => {
     window.location.assign('/login')
   }
 
-  return isLoggedIn && <a href="#" onClick={logout}>Logout</a>
+  return (isLoggedIn && <a href="#" onClick={logout}>Logout</a>)
 }
 
-export default LogoutButton
+export { LogoutButton }
```

Comment:
the export default in both cases are meant to keep a single component in each of these files. I don't think is the cause of the problem you are having nor an argument to change it at this point...